toc.yml

```diff
@@ -172,10 +172,6 @@
       - name: Deploying the .NET Framework
         tocHref: /dotnet/articles/framework/deployment/
         topicHref: /dotnet/articles/framework/deployment/index
-        items:
-        - name: Installing the .NET Framework on Windows
```

Comment:
on the same entry for the install folder, we should change this to Installation Guide